test/functional/ui/statusline_spec.lua

```diff
@@ -124,6 +124,56 @@ describe('global statusline', function()
     command('set ruler')
   end)
 
+  describe("clicks", function()
+    before_each(function ()
+      exec([=[
+        function! MyClickFunc(minwid, clicks, button, mods)
+          let mods = trim(a:mods)
+          if mods ==# ''
+            let g:testvar = printf("%d %d %s", a:minwid, a:clicks, a:button)
+          else
+            let g:testvar = printf("%d %d %s %s", a:minwid, a:clicks, a:button, mods)
+          endif
+        endfunction
+      ]=])
+    end)
+
+    it("works with vertical splits", function()
```

Comment:
please move this to the `describe('statusline clicks' ...` block instead of creating a new describe() block. The before_each you added also looks identical to the one there, so don't need it...

Reply:
Updated.